Formula/gpg-tui.rb

```diff
@@ -0,0 +1,33 @@
+class GpgTui < Formula
+  desc "Manage your GnuPG keys with ease! 🔐"
+  homepage "https://github.com/orhun/gpg-tui"
+  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "eccfaa3089605bb6df2cc49df208a93cc452d8a819b444bf82dfeb90739135ea"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "gnupg"
+  depends_on "gpgme"
+  depends_on "libgpg-error"
+  depends_on "libxcb"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    completions = share/"completions"
+    mkdir_p completions
+    ENV["OUT_DIR"] = completions
+    system bin/"completions"
+    bash_completion.install completions/"gpg-tui.bash"
+    fish_completion.install completions/"gpg-tui.fish"
+    zsh_completion.install completions/"_gpg-tui"
+
+    rm_rf completions
+    rm_f bin/"completions"
+    rm_f Dir[prefix/".crates*"]
+  end
+
+  test do
+    assert_match "gpg-tui [FLAGS] [OPTIONS]", shell_output("#{bin}/gpg-tui --help 2>&1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

----

I understand this is a TUI, but perhaps there are other flags we can pass to `gpg-tui` that can simulate some functionality. Otherwise, we can spawn a `PTY` to simulate interaction with it.

Reply:
You might want to take a look at `nnn` or `asciiquarium`. You could even use `grep "PTY"` to search in the Formula directory.

Reply:
Thanks for the suggestions, @nandahkrishna , based on the test for `zenith` I have come up with something that seems to work.